zebra-consensus/src/chain/tests.rs

```diff
@@ -102,6 +104,84 @@ fn verifiers_from_network(
     verifiers_from_checkpoint_list(network, CheckpointList::new(network))
 }
 
+static BLOCK_VERIFY_TRANSCRIPT1: Lazy<Vec<(Arc<Block>, Result<BlockHeaderHash, TransError>)>> =
```

Comment:
Issue: Use names that are specific, and different from other names.

```suggestion
static BLOCK_VERIFY_TRANSCRIPT_GENESIS: Lazy<Vec<(Arc<Block>, Result<BlockHeaderHash, TransError>)>> =
```